kuma/settings/common.py

```diff
@@ -1203,18 +1197,13 @@ def pipeline_one_scss(slug, **kwargs):
 
 if MAINTENANCE_MODE:
     # In maintenance mode, we're going to avoid using the database, and
-    # use Celery's default beat-scheduler as well as memcached for storing
+    # use Celery's default beat-scheduler as well as Redis for storing
     # any results. In both normal and maintenance mode we use djcelery's
     # loader (see djcelery.setup_loader() above) so we, among other things,
     # acquire the Celery settings from among Django's settings.
     CELERYBEAT_SCHEDULER = 'celery.beat.PersistentScheduler'
-    DEFAULT_CELERY_RESULT_BACKEND = (
-        'cache+memcached://' + ';'.join(
-            config('MEMCACHE_SERVERS',
-                   default='127.0.0.1:11211',
-                   cast=Csv())
-        )
-    )
+    DEFAULT_CELERY_RESULT_BACKEND = config('REDIS_CACHE_SERVER')
```

Comment:
This needs the same default as above. Maybe it would be better to write it as:

```
DEFAULT_CELERY_RESULT_BACKEND = CACHES['default']['LOCATION']
```